src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -660,27 +660,7 @@ public override string ToString()
 
         public override bool Equals(object obj)
         {
-            if (obj is Color)
-            {
-                Color right = (Color)obj;
-                if (value == right.value
-                    && state == right.state
-                    && knownColor == right.knownColor)
-                {
-                    if (name == right.name)
-                    {
-                        return true;
-                    }
-
-                    if (ReferenceEquals(name, null) || ReferenceEquals(right.name, null))
-                    {
-                        return false;
-                    }
-
-                    return name.Equals(name);
-                }
-            }
-            return false;
+            return obj is Color && this == (Color)obj;
```

Comment:
> I didn't look at the type, just at the 'deleted' code.

Look at this line:
https://github.com/dotnet/corefx/pull/12714/files/e4bad4b00780174ef7b23a2620d5d3accee9dd5e#diff-167862f543287597f9cd8ac6708f09bdL680